baselines/splunk_risky_command_abuse_ml_detection_baseline.yml

```diff
@@ -0,0 +1,71 @@
+name: Splunk Risky Command Abuse ML Detection Baseline
+id: 273df2f7-643a-451a-8d4d-637e39eadc87
+version: 1
+date: '2022-05-27'
+author: Abhinav Mishra, Kumar Sharad and Xiao Lin, Splunk
+type: Baseline
+datamodel:
+- Splunk_Audit
+description: 'This search is to build a per user risky command abuse detection model using MLTK DensityFunction 
+  algorithm based on audit log data of Splunk apps. The model is built from past 7 days users history of running 
+  search commands containing risky keywords, and aggregate the total search run time for each hour as indicator of 
+  user behavior. The default threshold of outlier boundary is defined as 0.1%. This search should be scheduled to 
+  run at least as frequently as every 7 days to build a recent baseline for each user. The name of ML model 
+  generated is "risky_command_abuse" and should be configured as global (not private) in MLTK app, unless the same 
```

Comment:
please rename "The name of ML mode generated is "risky_command_abuse" and should be configured as global (not private) in MLTK app, unless the same account of training this model will be used to perfeorm inference using this model for anomaly detection."

to 

"The name of ML mode generated is "risky_command_abuse". The model should be configured to be globally shared (not private) in the MLTK app as documented here: https://docs.splunk.com/Documentation/MLApp/5.3.1/User/Models#Sharing_models_from_other_Splunk_apps"

Reply:
done